app/routes/auth.py

```diff
@@ -16,9 +16,12 @@ def register():
     form = RegistrationForm()
     if form.validate_on_submit():
         hashed_password = bcrypt.generate_password_hash(form.password.data).decode('utf-8')
-        user = User(email=form.email.data, password=hashed_password, army_name=form.army_name.data)
+        user = User(email=form.email.data, password=hashed_password)
         db.session.add(user)
         db.session.commit()
+        army = Army(user_id=user.id)
```

Comment:
What about the army name? If it's a regular login, it means they put the army name in the form

Reply:
yea i made a mistake there, thanks